common/include/Utilities/IniInterface.h

```diff
@@ -70,10 +70,11 @@ class IniInterface
 	void EnumEntry( const wxString& var, T& value, const wxChar* const* enumArray=NULL, const T defvalue=(T)0 )
 	{
 		int tstore = (int)value;
+		int defaultvalue = (int)defvalue;
```

Comment:
Well, I don't like all those verbose cast, beside it is error-prone on the type.

First, we (you ;) ) need to add a function to ease the cast. Here some example from stack overflow. For the naming, I took `as_int` but there are various possibillities.
```
template <typename Enumeration>
auto as_int(Enumeration const value)
    -> typename std::underlying_type<Enumeration>::type
{
    return static_cast<typename std::underlying_type<Enumeration>::type>(value);
}

template <typename E>
constexpr typename std::underlying_type<E>::type as_int(E e) {
    return static_cast<typename std::underlying_type<E>::type>(e);
}

template <typename E>
constexpr auto as_int(E e) noexcept
{
    return static_cast<std::underlying_type_t<E>>(e);
}
```

Then EnumEntry uses a template type. However you will always cast to an `int` which defeat the purpose of the template. Boolean variable will be stored as int. I'm pretty sure we can find a better solution (SFINAE), basically a template that call EnumEntry(..., as_int(value), ....)